### PR TITLE
Do not include '../bits/..' system headers

### PR DESCRIPTION
**Pull request by stahlbauer**

### CODE REVIEW

File: src/goto-instrument/dump_c.cpp [hunk 1]
```diff
@@ -876,11 +876,12 @@ bool dump_ct::ignore(const symbolt &symbol)
...
   else if(!system_library_map.empty() &&
-          has_prefix(file_str, "/usr/include/") &&
-          file_str.find("/bits/")==std::string::npos)
+          has_prefix(file_str, "/usr/include/"))
   {
-    system_headers.insert(
-      file_str.substr(std::string("/usr/include/").size()));
+    if (file_str.find("/bits/")==std::string::npos) {
+      // Do not include transitive includes of system headers!
+      system_headers.insert(file_str.substr(std::string("/usr/include/").size()));
```
Comment (tautschnig) [inline src/goto-instrument/dump_c.cpp]: This line might be too long (80 chars max).

File: src/goto-instrument/dump_c.cpp [hunk 2]
```diff
@@ -876,11 +876,12 @@ bool dump_ct::ignore(const symbolt &symbol)
     return true;
   }
   else if(!system_library_map.empty() &&
-          has_prefix(file_str, "/usr/include/") &&
-          file_str.find("/bits/")==std::string::npos)
+          has_prefix(file_str, "/usr/include/"))
   {
-    system_headers.insert(
-      file_str.substr(std::string("/usr/include/").size()));
+    if (file_str.find("/bits/")==std::string::npos) {
```
Comment (tautschnig) [inline src/goto-instrument/dump_c.cpp]: Just style changes: No whitespace after `if`, and `{` should go on a new line.

**Review by tautschnig — CHANGES_REQUESTED** [review]
Some whitespace changes, please.

**Comment by stahlbauer** [comment]
The repository with the origin of pull request was broken. Closing this pull request and opening a new one.

**Pull request closed without merging**